iOS/DuckDuckGo/AppServices/DBPService.swift

```diff
@@ -25,10 +25,13 @@ import BrowserServicesKit
 import PixelKit
 import Networking
 
-final class DBPService: NSObject {
-
+final class DBPService: NSObject, EmailConfirmationDataChecking {
     private let dbpIOSManager: DataBrokerProtectionIOSManager?
 
+    public var emailConfirmationDataService: EmailConfirmationDataServiceProvider? {
```

Comment:
What's this for?

Reply:
Nvm, this has been removed as `EmailConfirmationDataChecking` has been removed in favor of `EmailConfirmationDataDelegate`